src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -2707,4 +2686,17 @@ export default class Aircraft {
     removeConflict(other) {
         delete this.conflicts[other.getCallsign()];
     }
+
+    /**
+     * Aircraft has "weight-on-wheels" (on the ground)
+     * @for AircraftInstanceModel
+     * @method wow
+     */
+    wow() {
+        const error_allowance = 5;
+        const apt = window.airportController.airport_get();
+        const rwy_elev = apt.getRunway(this.rwy_dep || this.rwy_arr).elevation;
+        const apt_elev = apt.position.elevation;
+        return this.altitude - (rwy_elev || apt_elev) < error_allowance;
```

Comment:
@n8rzz I come from an ATC background, where (basically) the only stuff worth discussing is how you should have worked your traffic differently, or done x/y/z better/faster/etc. So I appreciate the sentiment, but relative to what I work in, this is *far* from the negativity I'm already used to anyway; so no, you haven't been "harsh" at all.

Regardless, I'm glad to get such thorough feedback, especially when you often provide explanations as to *why* a particular change is needed. I've never been "set in my ways", and with each step, I'm coming to see why you've made each suggestion. Most important to me is improvements upon the sim, and it's getting cleaner and easier to work with every day, so you'll hear no complaints from this guy...